SOC/common/simulation.py

```diff
@@ -216,6 +216,26 @@ def animate(i):
             display(HTML(anim.to_html5_video()))
         else:
             return anim
+
+    def save(self, file_name = 'sim'):
+        """ serialization of object and saving it to file"""
+
+        root = zarr.open_group('state/' + file_name + '.zarr', mode = 'w')
```

Comment:
Byłoby fajnie, gdyby można było to połączyć z nazewnictem z #57. Chyba wiem, jak przy tym pomóc...